benchmark/profile_throughput.py

```diff
@@ -257,16 +260,25 @@ def main(dataset: str,
         csv (str, optional): The path to save the result.
         log_level(str, optional): The log level. Defaults to INFO
         seed (int, optional): Seed used in sampling prompts from dataset. Defaults to 0.
+        cache_count (float, optional): The ratio of k/v cache memory. Defaults to 0.5.
+        model_format (str, optional): The layout of the deployed model.
+            Its value should be among ['hf', 'llama', 'awq', None]. Defaults to hf
     """    # noqa
     random.seed(seed)
     os.environ['TM_LOG_LEVEL'] = log_level
 
-    engine = Engine(model_path,
-                    tp=tp,
-                    top_k=top_k,
-                    top_p=top_p,
-                    temperature=temperature,
-                    csv=csv)
+    group_size = 128 if model_format == 'awq' else 0
+    engine_config = TurbomindEngineConfig(model_name='llama',
+                                          session_len=4096,
+                                          model_format=model_format,
+                                          group_size=group_size,
```

Comment:
TurbomindEngineConfig does not have `group_size`
https://github.com/InternLM/lmdeploy/blob/4cc59fad2e626a20f4cfa9c5e630e5e4c33e2084/lmdeploy/messages.py#L108-L120